### ISSUE TITLE
Sideloadable IPA?

### ISSUE THREAD

**Issue by jarvisnexus** [rcastriotta/Stremio-iOS#3]
Hi @rcastriotta 

Is the development is already completed or it’s still under development? Can we have sideloadable IPA?

**Comment by qbap** [comment]
You can find the [Stremio IPA for iOS here](https://onejailbreak.com/blog/stremio-ipa-ios/).

**Comment by jarvisnexus** [comment]
Thank you so much @qbap

**Comment by jarvisnexus** [comment]
@qbap It’s crashing..

**Comment by qbap** [comment]
> [@qbap](https://github.com/qbap) It’s crashing..

How did you install it?

**Comment by jarvisnexus** [comment]
SideStore.

https://sidestore.io/

**Comment by qbap** [comment]
I have used Sideloadly and it works.

**Comment by jarvisnexus** [comment]
Yes, I did get the IPA from the other Github issue and it worked for me as well.
